flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.configuration;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.annotation.docs.ConfigGroup;
+import org.apache.flink.annotation.docs.ConfigGroups;
+import org.apache.flink.configuration.description.Description;
+
+import static org.apache.flink.configuration.description.LinkElement.link;
+import static org.apache.flink.configuration.description.TextElement.code;
+import static org.apache.flink.configuration.description.TextElement.text;
+
+/**
+ * Config options for restart strategies.
+ */
+@PublicEvolving
+@ConfigGroups(groups = {
+	@ConfigGroup(name = "FixedDelayRestartStrategy", keyPrefix = "restart-strategy.fixed-delay"),
+	@ConfigGroup(name = "FailureRateRestartStrategy", keyPrefix = "restart-strategy.failure-rate")
+})
+public class RestartStrategyOptions {
+
+	public static final ConfigOption<String> RESTART_STRATEGY = ConfigOptions
+		.key("restart-strategy")
+		.noDefaultValue()
+		.withDescription(
+			Description.builder()
+				.text("Defines the restart strategy to use in case of job failures.")
+				.linebreak()
+				.text("Accepted values are:")
+				.list(
+					text("%s, %s, %s: No restart strategy.", code("none"), code("off"), code("disable")),
+					text(
+						"%s, %s: Fixed delay restart strategy. More details can be found %s.",
+						code("fixeddelay"),
+						code("fixed-delay"),
+						link("../dev/task_failure_recovery.html#fixed-delay-restart-strategy", "here")),
```

Comment:
Hmm good question, maybe we should simply not translate headings? I guess that we need to address this in a separate issue. I'll create a JIRA issue for it: https://issues.apache.org/jira/browse/FLINK-13909

Reply:
Can we just remove these links for now? There aren't any other instances of options linking to other pages of the flink documentation.
If the link doesn't work on the chinese page the link checker will likely throw a fit.

Reply:
Hmm, yes this would be the short term solution. However, I really think that these kind of links are super helpful for users. Hence, I'm a bit hesitant to simply remove them. Can we exclude anchor checks from the documentation link checker instead?

Reply:
Ok, so the best option atm would be to put into the Chinese documentation the English headings. Since I believe these links are quite helpful, I'd rather do this than keeping the Chinese documentation completely in Chinese. In the end, the English documentation should be the ground truth and not being affected by its translations.

Reply:
For reference: https://issues.apache.org/jira/browse/FLINK-13922